tests/test_queryselector.py

```diff
@@ -0,0 +1,301 @@
+"""Test queryselector set of methods.
+Pyppeteer2 uses functions which have following counterparts in original puppeteer node.js:
+puppeteer 	    pyppeteer2 	                pyppeteer2 shorthand
+Page.$() 	    Page.querySelector() 	    Page.J()
+Page.$$() 	    Page.querySelectorAll() 	Page.JJ()
+Page.$x() 	    Page.xpath() 	            Page.Jx()
+Page.$eval()    Page.querySelectorEval()    Page.Jeval()
+Page.$$eval()   Page.querySelectorAllEval() Page.JJeval()
+
+"""
+
+import pytest
+
+from syncer import sync
+
+from pyppeteer.errors import ElementHandleError
+from tests.conftest import chrome_only
+
+
+@chrome_only
+@sync
+async def test_page_Jeval_executes_js_func(isolated_page):
+    """Test Page().Jeval() aka `querySelectorEval()` method,
+    which executes JS function with an element which matches ``selector``
+
+    """
+    page = isolated_page
+    await page.setContent('<section id="testAttribute">43543</section>')
+    idAttribute_Jeval = await page.Jeval('section', "e => e.id")
```

Comment:
Minor thing, but I'd prefer not to mix camel/underscore in the same statement. Which you commit to doesn't really matter as long as it's consistent, but personally I find an underscore approach a little more readable:
```suggestion
    id_attribute_Jeval = await page.Jeval('section', "e => e.id")
```

Reply:
Well, confusing topic regarding pyppeteer in whole, as the repo prefers to use camelCases (to mimic original puppeteer), but test fixtures uses snake_case.
I'll try to correct to camelCases, as I believe it's more consistent with lib methods.

Reply:
Yeah, we use camelCase to maintain (almost) a 1:1 mirror of puppeteer's API. But within our tests it's not as big of a deal.